src/main/java/com/hms_networks/americas/sc/extensions/system/time/SCTimeUnit.java

```diff
@@ -11,19 +11,19 @@
  */
```

Comment:
Improper usage of this API class. This commit should be dropped and constants remain private